### PR TITLE
Vaccine product and manufacturer terminology improvements

### PR DESCRIPTION
**Pull request by masnick**
This PR adds in support for alternatives to CVX for identifying vaccines.

Additionally, it improves our support for identifying vaccine manufacturer.

For the full list of added value sets, and information on specifying manufacturer, please see http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/vaccine-tx-improvements/StructureDefinition-vaccination-credential-immunization.html

### CODE REVIEW

**Comment by masnick** [comment]
Note the URI for the AIR code system is quite long: `https://www.humanservices.gov.au/organisations/health-professionals/enablers/air-vaccine-code-formats`.

File: input/includes/immunization.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+Note manufacturer and lot number may be recorded together in vaccine records. To conform with this IG, manufacturer SHALL NOT appear in the `lotNumber` element; instead, Issuers SHALL apply the above conformance criteria to determine when and how manufacturer can be identified.
+
+#### CVX and MVX codes
+
+When vaccine and manufacturer are provided using US-centric terminology (CVX and MVX, respectively) for COVID-19 vaccinations, CDC [provides a list](https://www.cdc.gov/vaccines/programs/iis/COVID-19-related-codes.html) that includes "Sale Proprietary Name" (e.g., `Moderna COVID-19 Vaccine`). The "Sale Proprietary Name" or other trade name SHALL NOT be included in FHIR resources, but MAY be used by actors when producing human-readable representations of these resources.
+
+Note that as of May 2021, CVX implicitly identifies specific vaccine products and manufacturers **only for COVID-19**. For other diseases, CVX typically identifies just the target disease and vaccine type. Issuers that only populate `vaccineCode` with a CVX code therefore MAY provide a manufacturer separately for COVID-19 vaccines, and SHOULD provide a manufacturer
```
Comment (jmandel) [inline input/includes/immunization.md:130]:
```suggestion
Note that as of May 2021, CVX implicitly identifies specific vaccine products and manufacturers **only for COVID-19**. For other diseases, CVX typically identifies just the target disease and vaccine type. Issuers that only populate `vaccineCode` with a CVX code therefore SHOULD NOT populate a `manufacturer` for COVID-19 vaccinations, but MAY provide a manufacturer for non-COVID-19 vaccinations.
```

Looks like this sentence was truncated, and I'm trying to infer what it meant. I hope/expect we can strongly discourage ("SHOULD NOT") issuers from including an explicit manufacturer when there's already an implicit manufacturter known.

**Comment by p9g** [comment]
Use http://hl7.org/fhir/sid/mvxfor the MVX link in Immunization.manufacturer.identifier.system
https://terminology.hl7.org/NamingSystem-MVX.html is wrong and fix is in the works

File: input/includes/immunization.md (hunk 1 above)
Comment (masnick) [reply input/includes/immunization.md]: Thanks Josh -- this was the intent here.

**Pull request merged**